### ISSUE TITLE
Warn/Error on duplicate message Id.

### ISSUE THREAD

**Issue by zumoshi** [nicksnyder/go-i18n#332]
Assuming the following code:

```go
text, err := localizer.LocalizeMessage(&i18n.Message{
	ID:          "X",
	Other:       "Text 1",
})
// ...
// somewhere else
// ...
text, err := localizer.LocalizeMessage(&i18n.Message{
	ID:          "X",
	Other:       "Something else",
})
```

is result of a copy paste mishap (forgot to change the message Id), the `goi18n extract` happily eats/ignores the first message and only outputs the second one to the translation file.

Can we get an error or warning printed out, or at the very list have the extractor create comments for filename/line numbers above the translations in the toml file so we can notice that a certain id is mentioned multiple times and investigate?

**Comment by nicksnyder** [comment]
Yeah can improve this. Will make it an extraction error if you have the same ID defined with different content.
